### PR TITLE
Change dates from 2014 to 2015

### PR DESCRIPTION
**Pull request by AnanyaKumar**
@lavalamp These are the requested changes for the first PR I submitted (that Victor accepted and merged in). Thanks again for reviewing! The other 16 files were already using 2015.

### CODE REVIEW

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# Copyright 2014 The Kubernetes Authors All rights reserved.
+# Copyright 2015 The Kubernetes Authors All rights reserved.
```
Comment (lavalamp) [inline hack/local-up-cluster.sh]: These should only be 2015 in files that you added just now-- I'm pretty sure this file isn't one of those. :)

Comment (AnanyaKumar) [inline hack/local-up-cluster.sh]: I changed this file - should I still keep the date to 2014? Should I only change the date for files that I've _added_? Thanks!

Edit: Sorry I just saw that you commented on the other PR and said that I should only change the date for new files.

Comment (lavalamp) [inline hack/local-up-cluster.sh]: Yes, only file that you've added. Thanks :)

**Comment by lavalamp** [comment]
LGTM

**Pull request merged**